tools/eslint-plugin/package.json

```diff
@@ -38,8 +38,8 @@
     "eslint-plugin-jsdoc": "35.1.3",
     "eslint-plugin-jsx-a11y": "6.4.1",
     "eslint-plugin-prefer-arrow": "1.2.3",
-    "eslint-plugin-react": "7.24.0",
-    "eslint-plugin-react-hooks": "4.2.0",
+    "eslint-plugin-react": "^7.32.2",
```

Comment:
could you add ^ for the rest of these plugin deps?

@paulius-valiunas any reason these were all pinned?

Reply:
Okay

Reply:
shouldn't we remove this package now that we have https://github.com/iTwin/eslint-plugin/?

Reply:
Yes, but I am not sure when we plan on doing that.

Reply:
Need to get this into 3.7. I wasn't expecting any new changes to the pkg in 3.x. As soon as this pr completes we will nuke this PKG from the core repo